### PR TITLE
Playdigo: add user sync

### PR DESCRIPTION
**Pull request by Yanivplaydigo**

### CODE REVIEW

**Comment by Yanivplaydigo** [comment]
@bretg @onkarvhanumante @gargcreation1992 @bsardo @SyntaxNode @Sonali-More-Xandr please respond to this pull request

**Review by przemkaczmarek — APPROVED** [review]

File: static/bidder-info/playdigo.yaml [hunk 1]
```diff
@@ -14,3 +14,10 @@ capabilities:
     - banner
     - video
     - native
+userSync:
+  redirect:
+    url: "https://cs.playdigo.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
+  iframe:
+    url: "https://cs.playdigo.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
```
Comment (bsardo) [inline static/bidder-info/playdigo.yaml:22]: This endpoint does not appear to work. I'm getting a 404. I tested it with the following URL:
`https://cs.playdigo.com/pbserverIframe?gdpr=0&gdpr_consent=CPuESnSPuESnSPiAAAENCZCAAIAAAAAAAAAAAQAAAAAEAAAAAA&ccpa=&gpp=&gpp_sid=&pbserverUrl=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dplaydigo%26gdpr%3D0%26gdpr_consent%3DCPuESnSPuESnSPiAAAENCZCAAIAAAAAAAAAAAQAAAAAEAAAAAA%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3DabcUID`

Screenshot 2024-08-02 at 10 54 08 AM

Comment (Yanivplaydigo) [reply static/bidder-info/playdigo.yaml:22]: fixed

Comment (bsardo) [reply static/bidder-info/playdigo.yaml:22]: Confirmed iframe works
Screenshot 2024-08-06 at 10 41 39 AM

**Review by przemkaczmarek — APPROVED** [review]

**Pull request merged**